components/withBreadcrumbs.tsx

```diff
@@ -34,11 +34,13 @@ const WithBreadcrumbs: FC<WithBreadcrumbsProps> = ({ navKeys = [] }) => {
     return pathList.reduce((breadcrumbs, path, index) => {
       const nodeWithCurrentPath = currentNode.find(
         ([nodePath, entry]) =>
-          nodePath === path &&
+          // Checking link in cases where nodePath cannot = path. Like 'discoverJavaScriptTimers'
+          (nodePath === path || entry.link === pathname) &&
```

Comment:
What if we only check the entry.link? I fail to recall that we were comparing nodePath === path, to begin with. What are nodePath and path again?

Reply:
nodePath is from navigation.json and path is just pathname split with '/' and converted to camelCase

Reply:
not all paths have links, for example, 'learn' and 'gettingStarted' have empty string as link

Reply:
True, it's been a while since I implemented this, and many people have already updated/touched it before claiming to fix it. Hopefully this time it actually fixes it :)